modules/client/src/main/java/org/apache/ignite/client/IgniteClient.java

```diff
@@ -260,6 +263,23 @@ public Builder heartbeatTimeout(long heartbeatTimeout) {
             return this;
         }
 
+        /**
+         * Sets the SSL configuration.
+         *
+         * @param sslConfiguration SSL configuration.
+         * @return This instance.
+         */
+        public Builder ssl(@Nullable SslConfiguration sslConfiguration) {
+            if (sslConfiguration == null) {
+                this.sslConfiguration = SslConfiguration.builder().build();
```

Comment:
Let's just keep it null, I'm not sure why do we need a default instance in this case.